src/agent/agent.service.ts

```diff
@@ -18,8 +18,8 @@ export class AgentService {
         // One option is for this controller (at self url) to manage all the interactions,
         //   which would require a lot of overlap with the agency controller
         // The simpler option for right now is to use the agency's controller and specify a specific policy (eg issuer, verifier, etc)
-        // const controllerUrl = process.env.SELF_URL + '/v1/controller';
-        const controllerUrl = null;
+        // const controllerUrl = null;
+        const controllerUrl = process.env.SELF_URL + '/v1/controller';
```

Comment:
This is the only line that makes me nervous - where you able to use npm link to see if this works in our controllers? If not it may be best to comment it out until we have the demo controller integration tests in this repo